### PR TITLE
Fixes #3794

### LINKED ISSUE #3794: (WEB REPORT BY: zth REMOTE: 198.245.63.50:7777) span class='notice'>You add 1 chips...
>Revision (Should be above if you're viewing this from ingame!)

>General description of the issue
span class='notice'>You add 1 chips to your stack of cash.

>What you expected to happen
<span class='notice'>You add 1 chips to your stack of cash.

>What actually happened
span class='notice'>You add 1 chips to your stack of cash.

>Steps to reproduce if possible
attach chips

### PR DESCRIPTION
**Pull request by ghost**
- Fix broken span

While at it, I did some formatting work

### CODE REVIEW

File: code/game/objects/items/weapons/cash.dm [hunk 1]
```diff
@@ -35,22 +35,23 @@ var/global/list/moneytypes=list(
 	update_icon()
 
 /obj/item/weapon/spacecash/examine(mob/user)
-	if(amount>1)
+	if(amount > 1)
 		setGender(PLURAL)
 		..()
 		user << "It is a stack holding [amount] chips."
-		user << "<span class='info'>It's worth [worth*amount] credits.</span>"
```
Comment (clusterfack) [inline code/game/objects/items/weapons/cash.dm]: This is already correct though

File: code/game/objects/items/weapons/cash.dm [hunk 2]
```diff
@@ -35,22 +35,23 @@ var/global/list/moneytypes=list(
...
+		user << "<span class='info'>It is worth [worth*amount] credits.</span>"
 	else
 		setGender(NEUTER)
 		..()
+		user << "<span class='info'>It is worth [worth] [worth > 1 ? "credits":"credit"].</span>"
```
Comment (clusterfack) [inline code/game/objects/items/weapons/cash.dm]: Ya ever think I left this out on purpose?

**Comment by dylanstrategie** [comment]
Hold on a second, that looks like the PR missed something

File: code/game/objects/items/weapons/cash.dm [hunk 3]
```diff
@@ -43,14 +43,15 @@ var/global/list/moneytypes=list(
 	else
 		setGender(NEUTER)
 		..()
+		user << "<span class='info'>It's worth [worth] credit.</span>"
```
Comment (clusterfack) [inline code/game/objects/items/weapons/cash.dm]: This still isn't necessary, its already being described as a singular credit of some amount. There's no need to say, this is a 100 credit chip, it's worth 100 credit.

Comment (dylanstrategie) [inline code/game/objects/items/weapons/cash.dm]: Right then, I'll snap it off

**Pull request merged**